wirecard/oxid-ee
# WDPD-26 Logging payments Results

### This PR

* Revision of logging inside Paypal payment flow
* Slight changes of logging messages error result items

### How to Test

* Enable Paypal payment method
* As a customer: make order with Paypal payment method
* As an integrator: check logs within oxid logs

### Jira Links

* [https://jira.parkside.at/browse/WDPD-26]
